features/extraction.py

```diff
@@ -131,6 +158,11 @@ def create_config(
         config, post_config = rasr.build_config_from_mapping(
             crp, {"corpus": "extraction.corpus"}, parallelize=True
         )
+        post_config[
+            "*"
+        ].OUTPUT_DIR = (
+            "./"  # make path very unspecific to allow easy override via command-line
```

Comment:
```suggestion
            "."  # make path very unspecific to allow easy override via command-line
```
as the `/` will be added by `out_path = os.path.join(out_dir, out_path)`